android-sdk/src/main/java/org/sagebionetworks/bridge/android/manager/UploadManager.java

```diff
@@ -149,7 +147,8 @@ public Completable processUploadFiles() {
                                 processUploadForCachedSession(
                                         uploadFile,
                                         uploadDAO.getUploadSession(uploadFile.filename)
-                                )));
+                                ))).doOnError(t -> LOG.warn("Failed to process upload files"))
+                .onErrorComplete();
```

Comment:
This will prevent those upload failures from crashing the app. Also, those crashes weren't showing in logs at all, but the added logging now identifies the crash and makes it easier to follow the flow.

Reply:
I hate that the crash doesn't show up in logs, it's so confusing